operators/constellation-node-operator/main.go

```diff
@@ -196,6 +219,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	wg := &sync.WaitGroup{}
+	exec.Start(context.Background(), wg)
+	defer wg.Wait()
+	defer exec.Stop()
```

Comment:
Stop would make sure to only return when the Starter routines have finished
```suggestion
	waitFn := exec.Start(context.Background())
	defer exec.Stop(waitFn)
```